SwissKnife/gcloud/GCloudStorage.py

```diff
@@ -203,36 +216,38 @@ def __generate_blob(self,
 
         return blob
 
-    @staticmethod
-    def get_storage_complete_file_path(file_name: str=None,
-                                       file_path: str=None,
-                                       with_bucket: bool=False,
-                                       with_prefix: bool=True,
-                                       with_gs: bool=True) -> str:
+    def get_storage_complete_file_path(self,
+                                       file_name: str = None,
+                                       file_path: str = None,
+                                       with_bucket: bool = False,
+                                       with_prefix: bool = True,
+                                       with_gs: bool = True) -> str:
         """Returns the complete path of a file stored in gcloud.
 
         :param file_path: Path of the file, without bucket
         :type file_path: str
         :param file_name: Name of the file
         :type file_name: str
-        :param with_bucket: If the bucket will be returned, defaults to False
+        :param with_bucket: Whether bucket will be return or not, defaults to False
```

Comment:
Isn't "returned" the past tense of "return"?

Reply:
It is! my bad. I rephrased the sentence for clarity but I don't know if i made it